src/arrays/ArrayUtil.java

```diff
@@ -36,60 +69,99 @@ public static void create2dArray(int x, int y) {
      * @return true if the array contains the given value, otherwise return false.
      */
     public static boolean contains(int[] array, int value) {
-
+        for(int i=0;i<array.length-1;i++) {
+            if (array[i] == value) {
+                return true;
+            }
+        }
         return false;
     }
 
+
     /**
      * @param array An array to search in.
      * @param value The value to search for.
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
-
-        return 0;
+        for(int i=0;i<array.length-1;i++) {
+            if (array[i] == value) {
+                return i;
+            }
+        }
+        return -1;
     }
 
     /**
      * Remove the element by the given index.
      * Print new array
      */
     public static void remove(int[] array, int index) {
```

Comment:
The method always prints 0 for the last element in the array, please check it one more time